frame/pablo/src/lib.rs

```diff
@@ -122,6 +130,28 @@ pub mod pallet {
 			/// Amount of minted lp tokens.
 			mint_amount: T::Balance,
 		},
+		/// Liquidity added into the pool `T::PoolId`.
+		LiquidityAddedToLiquidityBootstrappingPool {
+			/// Pool id to which liquidity added.
+			pool_id: T::PoolId,
+			/// Amount of base asset deposited.
+			base_amount: T::Balance,
+			/// Amount of quote asset deposited.
+			quote_amount: T::Balance,
+		},
+		/// Liquidity added into the pool `T::PoolId`.
+		LiquidityAddedToConstantProductPool {
```

Comment:
LiquidityAdded events for ConstantProduct and StableSwap are same. And LBP can also use that same event without any issue.

Reply:
Idea is to keep it this way till reach cu-23v2y3n

Reply:
This change does not bring in LBP to pablo. So perhaps we can remove `LiquidityAddedToLiquidityBootstrappingPool` and just have `LiquidityAdded` which is used for both StableSwap and Uniswap